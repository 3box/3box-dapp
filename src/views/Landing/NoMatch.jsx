import React from 'react';

import Footer from './components/Footer';
import '../styles/Info.css';
import '../styles/Landing.css';

const NoMatch = () => (
  <div className="info NoMatchPage">

    <div className="info__page">
      <div className="info__banner--NoMatchPage">
        <h1>This page doesn't exist</h1>
      </div>
    </div>

    <Footer />
  </div>
);

export default NoMatch;
