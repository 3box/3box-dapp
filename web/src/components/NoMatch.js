import React from 'react';

export function NoMatch() {
  return (
    <h1>
404 not found
    </h1>
  );
}

export default NoMatch;
