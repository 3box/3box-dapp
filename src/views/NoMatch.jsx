import React from 'react';

import Footer from '../components/Footer';
import './styles/Info.css';
import './styles/Landing.css';

const NoMatch = ({ isLoggedIn, handleSignInUp }) => (
  <div className="info NoMatchPage">

    <div className="info__page">
      <div className="info__banner--NoMatchPage">
        <h1>This page doesn't exist</h1>
      </div>
    </div>

    <Footer
      handleSignInUp={handleSignInUp}
      isLoggedIn={isLoggedIn}
    />

  </div>
);

export default NoMatch;
