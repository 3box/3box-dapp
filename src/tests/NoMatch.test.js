import React from 'react';
import ReactDOM from 'react-dom';
import NoMatch from '../components/NoMatch';
import {
  renderWithIntl,
  rendererCreateWithIntl,
} from './helper';

test('renders without crashing', () => {
  const div = document.createElement('div');
  renderWithIntl(<NoMatch />, div);
  ReactDOM.unmountComponentAtNode(div);
});

test('render correctly', () => {
  const component = rendererCreateWithIntl(<NoMatch />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
