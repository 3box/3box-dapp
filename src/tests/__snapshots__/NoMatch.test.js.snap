// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<h1>
  404 not found
</h1>
`;
